test/hdnode.js

```diff
@@ -368,6 +405,30 @@ describe('HDNode', function () {
       assert.throws(function () {
         master.derive('foo')
       }, /Expected UInt32/)
+      assert.throws(function () {
+        master.derivePath()
+      }, /Expected Path/)
+      assert.throws(function () {
+        master.derivePath(2)
+      }, /Expected Path/)
+      assert.throws(function () {
+        master.derivePath([2, 3, 4])
+      }, /Expected Path/)
+      assert.throws(function () {
+        master.derivePath('/')
+      }, /Expected Path/)
+      assert.throws(function () {
+        master.derivePath('m/m/123')
+      }, /Expected Path/)
```

Comment:
@runn1ng if you're time is running short on this (not that there has been any hint of that),  I'll see if I can submit a PR to this branch (on your repo) to help out :)